include/constants/global.h

```diff
@@ -101,6 +101,7 @@
 
 // string lengths
 #define ITEM_NAME_LENGTH 14
+#define ITEM_NAME_PLURAL_LENGTH ITEM_NAME_LENGTH + 2 // the longest plural name (ElectrcMemories) is ITEM_NAME_LENGTH + 1
```

Comment:
If the longest is ITEM_NAME_LENGTH + 1, why is this ITEM_NAME_LENGTH + 2?

Reply:
Because y is suffixed to ies when it's plural eg Berry to Berries and this should still be working wrt custom items using max item name length IMO. That's +2 additional characters in length.

Reply:
I agree, but there's just a discrepancy between the comment and the actual value